src/geode/mesh/helpers/rasterize.cpp

```diff
@@ -466,16 +466,12 @@ namespace
         const geode::Segment3D& segment,
         const std::array< geode::GridCellsAroundVertex3D, 2 > /*unused*/ )
     {
-        constexpr auto cell_tested_attribute_name = "tested_cell_rasterize";
         auto cells = geode::rasterize_segment( grid, segment );
-        auto tested_att =
-            grid.cell_attribute_manager()
-                .find_or_create_attribute< geode::VariableAttribute, bool >(
-                    cell_tested_attribute_name, false );
+        std::vector< geode::index_t > tested_cells( grid.nb_cells(), false );
         std::queue< geode::GridCellIndices3D > to_test;
         for( const auto& cell : cells )
         {
-            tested_att->set_value( grid.cell_index( cell ), true );
+            tested_cells[grid.cell_index( cell )] = true;
             for( auto&& neighbor : neighbors( grid, cell ) )
             {
                 to_test.emplace( neighbor );
```

Comment:
```suggestion
                to_test.emplace( std::move( neighbor ) );
```
I read recently that this move is still required to ensure a rvalue is passed and not an on-the-fly new lvalue